### DISCUSSION TITLE
Partials/templating documentation?

### DISCUSSION THREAD

**Discussion by KruseCtrl** [retypeapp/retype#433] [Q&A]
On the [roadmap page](https://retype.com/roadmap/#v200) it appears that templating has been added in. There is one example of how to use templating for an alert message in the discussion#213, but other than that the only documentation for it is how to enable/disable it for a project or page.

Is there any more documentation for using templating? if so, where?

**Comment by KruseCtrl** [comment]
In the [discussion](https://github.com/retypeapp/retype/discussions/213) I mentioned, the documentation for the engine behind templating: Scriban is linked. So there is documentation for how to use some templating on a page.

However, the project config syntax given as an example didn't work for me. After some more digging, I found the correct Project variable syntax in [discussion 372](https://github.com/retypeapp/retype/discussions/372)

```
data:
    uuid: "123456"
```
Putting this into my retype.yml file means that i can have project wide variables, or putting this into the metadata section of a page gives a page wide variable, but i haven't been able to get a folder wide variable yet. Putting the same syntax into an index.yml doesn't quite work.

**Reply by geoffreymcgill** [reply]
Currently, only Page level and Project level custom `data` variables are supported. Folder level variables are not supported, but that is an interesting idea and I don't think something we thought about previously. It's something we might be able to get working in a future release.
